api/v1/routes/invitations.py

```diff
@@ -42,4 +43,27 @@ async def add_user_to_organization(
 
     logging.info(f"Processing invitation ID: {invite_id}")
 
-    return invite.InviteService.add_user_to_organization(invite_id, session)
\ No newline at end of file
+    return invite.InviteService.add_user_to_organization(invite_id, session)
+
+@invites.delete("/{invite_id}", status_code=200, response_model=success_response)
+def delete_invite(
+    invite_id: str,
+    db: Session = Depends(get_session), 
+    admin: User = Depends(user_service.get_current_super_admin)
+):
+    """ Delete invite from database """
+    if invite_id.strip() == "":
```

Comment:
Please stop this thing. It is not needed

Reply:
> Please stop this thing. It is not needed

Changes implemented.